CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [1.1.1] - 2017-06-12
```

Comment:
Since we're adding a new dependency, metrics-graphite, it would probably be good to make this a Y bump, 1.2.0.  We've been trying to do that for semver.